components/SignUpForm.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { useRouter } from "next/router";
+import axios from "axios";
+import { EmailInput, emailCheck } from "./EmailInput";
+import { SignUpPassword, isValidPassword } from "./SignUpPassword";
+import styles from "@/styles/signup.module.scss";
+import classNames from "classnames/bind";
+
+export function SignUp() {
+  const cx = classNames.bind(styles);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [emailError, setEmailError] = useState("");
+  const router = useRouter();
+
+  const signUp = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const emailChecked = emailCheck(email);
+    const validPassword = isValidPassword(password);
```

Comment:
emailCheck , isValidPassword 둘 다 동일하게 유효성 검증을 하는 함수인데 
네이밍 방식도 다르고 사용한 단어 (valid, check) 도 달라서 좀 헷갈리는 것 같아요 

멘토링 때 말씀드린대로 함수는 동사로 시작하게 작성해보시면 좋을 것 같고,
동일한 역할을 하는 함수라면 동일한 단어를 사용해서 일관성 있게 작성하면 더 좋을 것 같습니다! 

함수의 리턴값이 boolean 형이니 변수도 boolean 형에 맞게 지어주시면 좋을 것 같고요! 

```suggestion
    const isEmailValid = validateEmail(email);
    const isPasswordValid = validatePassword(password);
```